app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardFragment.kt

```diff
@@ -202,11 +183,10 @@ class DashboardFragment : BaseFragment(R.layout.fragment_dashboard) {
         /**
          * If the user starts the app for the first time the service will be started automatically
          */
-        // TODO: 27/05/2020 dusanjencik: Decide if we want to enable it automatically or not
-//        if (viewModel.wasServiceEnabledAutomaticallyOnFirstStart.not()) {
-//            viewModel.wasServiceEnabledAutomaticallyOnFirstStart = true
-//            checkDependenciesAndStartAutomaticHandshake(true)
-//        }
+        if (viewModel.wasServiceEnabledAutomaticallyOnFirstStart.not()) {
```

Comment:
wasServiceEnabledAutomaticallyOnFirstStart needs to be renamed. We don´t have no service anymore

Also: I´m unsure if this is a valid operation, unsure if UX approved to register the app automatically for Exposure notifications.

Reply:
Renamed https://github.com/austrianredcross/stopp-corona-android/pull/83#discussion_r433731286